CoolApp/CoolApp/AppDelegate.h

```diff
@@ -1,9 +1,9 @@
 //
 //  AppDelegate.h
-//  EmbeddedRuntime
+//  CoolApp
 //
-//  Created by Yavor Georgiev on 4.01.16 г..
-//  Copyright © 2016 г. NativeScript. All rights reserved.
```

Comment:
What is the reason for changing the Copyright and the creation header of this file?
As far as I see there are no changes made in file except this.

Reply:
Because that is what XCode does when you create a 100% brand new app. :grinning:  The appDelegate.h/c code that is in this app is just 100% stock XCode generated.     I know in my full recreations of the app (which I did twice, once to figure it all out, and once to follow my howto instructions to make sure the howto worked).   I didn't need to make any modifications to the AppDelegate, so it should be 100% what XCode generated .    But since XCode created everything on my machine it got my copyright.    Later when we decided which repo to stick it in, Git was "smart" -- it realized they were basically the same file so apparently git treated mine as a patch and all that changes was the copyright in those files.

Reply:
I should also state, if Progress wants the copyright to this; I have no issues with Progress having it.   :grinning:   Just make sure my name is somewhere on the project is all I request, so I can point to this as some of the technical type work I can do.    :grinning: